src/modules/CreateChannel/components/InviteUsers/index.tsx

```diff
@@ -91,10 +91,12 @@ const InviteUsers: React.FC<InviteUsersProps> = ({
       titleText={titleText}
       submitText={submitText}
       type={ButtonTypes.PRIMARY}
-      disabled={Object.keys(selectedUsers).length === 0}
+      disabled={users.length > 0 && Object.keys(selectedUsers).length === 0}
```

Comment:
This logic stil disable the submit button when there's no selected user

Reply:
Right. This part is related to this conversation https://sendbird.slack.com/archives/C050Z6DA70A/p1688089591024089?thread_ts=1688086902.570209&cid=C050Z6DA70A if I got your point correctly